### PR TITLE
chore: Unit testing on minimap position

### PR DESCRIPTION
**Pull request by cesarades**
**Description**
Added unit tests for the positioning of the minimap in the workspace. The test covers all 8 of the workspace configurations  (all permutations of LTR/RTL, Vertical/Horizontal, Start/End). I also included a test for when the scrollbar is not there.

### CODE REVIEW

File: plugins/workspace-minimap/src/positioned_minimap.ts
Comment (gonfunko) [inline plugins/workspace-minimap/src/positioned_minimap.ts]: What exception(s) are you catching here?

File: plugins/workspace-minimap/test/minimap_tests.mocha.js
Comment (gonfunko) [inline plugins/workspace-minimap/test/minimap_tests.mocha.js]: If the toolbox is at the top, would it make more sense for the minimap to be in the bottom left corner?

File: plugins/workspace-minimap/test/minimap_tests.mocha.js
Comment (gonfunko) [inline plugins/workspace-minimap/test/minimap_tests.mocha.js]: Throughout, WDYT about doing 

```
Object.assign(this.options, {
  RTL: false,
  horizontalLayout: false,
  toolboxPosition: start
});
```

I feel like that might make the fields that are being changed a bit clearer by removing the this.options. boilerplate?

File: plugins/workspace-minimap/src/positioned_minimap.ts
Comment (cesarades) [reply plugins/workspace-minimap/src/positioned_minimap.ts]: This is put in place to make unit testing possible. That function triggers an error due to the way the unit test is designed.

File: plugins/workspace-minimap/test/minimap_tests.mocha.js
Comment (cesarades) [reply plugins/workspace-minimap/test/minimap_tests.mocha.js]: I could definitely change that. I just wanted to keep it consistent with the backpack plugin.

File: plugins/workspace-minimap/test/minimap_tests.mocha.js
Comment (cesarades) [reply plugins/workspace-minimap/test/minimap_tests.mocha.js]: Love that

File: plugins/workspace-minimap/src/positioned_minimap.ts
Comment (cesarades) [reply plugins/workspace-minimap/src/positioned_minimap.ts]: Update:
To fix this I split the function. Now the unit test doesn't have to worry about the setMinimapCss() method.

File: plugins/workspace-minimap/test/minimap_tests.mocha.js
Comment (BeksOmega) [inline plugins/workspace-minimap/test/minimap_tests.mocha.js]: Why `parseInt`?

**Review by BeksOmega — APPROVED** [review]
This LGTM =) but wait for Aaron's approval as well!

File: plugins/workspace-minimap/test/minimap_tests.mocha.js
Comment (cesarades) [reply plugins/workspace-minimap/test/minimap_tests.mocha.js]: Because the result is actually 831.66666666666

Comment (gonfunko) [reply plugins/workspace-minimap/test/minimap_tests.mocha.js]: Math.round() might make the intent a bit clearer.

**Review by gonfunko — APPROVED** [review]

**Pull request merged**
